docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "2.2"
```

Comment:
In prod my plans are to use RDS not run our own postgres instance so would having the db dockerised like this provide us with any value?

Reply:
absolutely, you can make your own clusters now and get an environment very similar to production. Plus this ensure we have a consistent environment at all time, especially for other contributors. It should never be required to install and configure extra software to run a project (this is how we get it runs of my computer excuses!)

Reply:
Nice! Gotcha